portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsRulesActions.testcase

```diff
@@ -287,6 +287,95 @@ definition {
 		FormFields.viewRequiredAsteriskDataFieldNotPresent(fieldFieldLabel = "Numeric");
 	}
 
+	@description = "LPS-71592 - Verify that a Rule with 'Calculate' action can be created"
+	@priority = "5"
+	test CanCreateACalculateRule {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
```

Comment:
@leticiachiu, is the `property test.name.skip.portal.instance = "FormRules#"CanCreateACalculateRule;` missing intentionally? Can you please clarify?

Reply:
Yes, because me and Diogo are testing if the property is essential to the test and wait to fail (or not).. but I can put it back too